pages/inactiveCards.jsx

```diff
@@ -0,0 +1,9 @@
+const inactiveCards = () => {
+  return (
+    <>
+      <section>
+        <p>inactiveCards</p>
+      </section>
+    </>
+  );
+};
```

Comment:
This component looks exactly like the `browsByCategory` component. Maybe you can get rid of one of the component?

Reply:
Thanks for your review - Much appreciated!
I'll implement all suggested improvements with the next commit.